package.json

```diff
@@ -6,6 +6,9 @@
     "@testing-library/jest-dom": "^4.2.4",
```

Comment:
there is a redundant folder named `The-Act-of-Time` that got created, please get rid of that.
I'm adding eyal's explaination to it:
![image](https://user-images.githubusercontent.com/67635802/108096848-1ed4ea80-708a-11eb-8b50-f6c29f7e2218.png)